pkg/test/runner/runner.go

```diff
@@ -117,7 +117,11 @@ func (r *Runner) runFnEval() error {
 		kptArgs = append(kptArgs, "--exec-path", r.testCase.Config.EvalConfig.ExecPath)
 	}
 	if r.testCase.Config.EvalConfig.FnConfig != "" {
-		kptArgs = append(kptArgs, "--fn-config", r.testCase.Config.EvalConfig.FnConfig)
+		fnConfigPath, err := filepath.Abs(filepath.Join(r.testCase.Path, r.testCase.Config.EvalConfig.FnConfig))
```

Comment:
I am assuming you are special casing this for the test, but `kpt fn eval` does this if someone specifies relative path `.../fn-config.yaml` on the commandline ?

Reply:
I am not sure I understand your question. 

`fn eval` will not convert the relative fn config path to absolute because the path is relative to the kpt working directory and it's meaningless to do this kind of conversion. However, in tests, the directory which contains the config path is irrelevant to the kpt working directory so we cannot directly use relative path here. 

So yes, this is special for tests.